skillmap/src/components/AppModal.tsx

```diff
@@ -169,9 +169,17 @@ export class AppModalImpl extends React.Component<AppModalProps, AppModalState>
     }
 
     renderCodeCarryoverModal() {
-        const  { dispatchHideModal, skillMap, activity, pageSourceUrl, userState, dispatchSetReloadHeaderState } = this.props;
+        const  { skillMap, activity, pageSourceUrl, userState, dispatchHideModal, dispatchSetReloadHeaderState } = this.props;
         const carryoverModalTitle = lf("Keep code from previous activity?");
-        const carryoverModalText = lf("Do you want to start with your code from the previous activity or start fresh with starter code? Your images, tilemaps, tiles, and animations will stick around either way.");
+        const carryoverModalText = lf("Do you want to start with your code from {0} \
+            or start fresh with starter code? Your images, tilemaps, \
```

Comment:
Huh, I hadn't realized escaping new lines in strings was supported in js; does the spacing come out okay? / do we extract it & upload properly to crowdin? I'd expect it to have an extra three tabs of spaces between each line

Reply:
ah, i hadn't thought of crowdin, good call--will update!